### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by vishnubhagwan**
@josegonzalez ... please review.

### CODE REVIEW

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -31,7 +31,9 @@ The installation process takes about 5-10 minutes, depending upon internet conne
 
 Once the installation is complete, you can open a browser to setup your SSH key and virtualhost settings. Open your browser of choice and navigate to the host's IP address - or the domain you assigned to that IP previously - and configure Dokku via the web admin.
 
-*If you don't complete setup via the web installer (even if you set up SSH keys and virtual hosts otherwise) your Dokku installation will remain vulnerable to anyone finding the setup page and inserting their key.*
+```diff
+-If you don't complete setup via the web installer (even if you set up SSH keys and virtual hosts otherwise) your Dokku installation will remain vulnerable to anyone finding the setup page and inserting their key.
```
Comment (josegonzalez) [inline docs/getting-started/installation.md]: Can you make this a blockquote instead of a codeblock?

Comment (josegonzalez) [inline docs/getting-started/installation.md]: Also prefix with `Warning:`

Comment (vishnubhagwan) [inline docs/getting-started/installation.md]: Yeah sure..

**Comment by vishnubhagwan** [comment]
Can you please suggest the changes for me

File: docs/getting-started/installation.md [hunk 2]
```diff
@@ -31,9 +31,7 @@ The installation process takes about 5-10 minutes, depending upon internet conne
 
 Once the installation is complete, you can open a browser to setup your SSH key and virtualhost settings. Open your browser of choice and navigate to the host's IP address - or the domain you assigned to that IP previously - and configure Dokku via the web admin.
 
-```diff
--If you don't complete setup via the web installer (even if you set up SSH keys and virtual hosts otherwise) your Dokku installation will remain vulnerable to anyone finding the setup page and inserting their key.
-```
+>>**Warning: If you don't complete setup via the web installer (even if you set up SSH keys and virtual hosts otherwise) your Dokku installation will remain vulnerable to anyone finding the setup page and inserting their key.**
```
Comment (josegonzalez) [inline docs/getting-started/installation.md]: Should just be a single `>`. As well, you can place the `**` around _just_ the `Warning` prefix. I can then cleanup the js so this looks nice :)

Comment (vishnubhagwan) [inline docs/getting-started/installation.md]: #2437  please review the changes...

**Comment by josegonzalez** [comment]
Thanks for the pull request!

**Pull request merged**